### PR TITLE
Hotfix/suit 8555 demo set up for js api with mocha and some minor changes

### PR DESCRIPTION
**Pull request by AntonSBilan**

### CODE REVIEW

**Comment by andriichumak** [comment]
@AntonSBilan This PR contains changes for lib as well. Some of those changes has their own tickets. This will not do.

Please, separate following changes to their own PRs:
* call to closeSession
* default concurrency
* any other changes outside of "demo/" folder

I'll assign corresponding tickets to you.

File: demo/readme.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Demo set up for Suitest JS API with Mocha test runner
 
-1. Copy `testConfig.js.dist` to `testConfig.js`.
-2. Fill in fields with necessary data.
-3. Set automatedSession to `true` if you want to start test pack and run test in automated session mode.
-4. Update or remove `test/dummy.test.js`, optionally create more tests in that folder.
-5. In terminal navigate to `demo` folder and run `yarn` or `npm i`.
-6. Run `yarn test` or `npm test`.
+1. You may want create config file instead of passing params directly in console
```
Comment (andriichumak) [inline demo/readme.md]: 1. Copy `.suitestrc.dist` file as `.suitestrc` and fill in it's contents with your values according to [documentation](https://suite.st/docs/suitest-api/setup/#environment-setup).
2. Write your tests in `test/` folder. There is an example in `test/dummy.test.js`.

File: demo/test/dummy.test.js [hunk 2]
```diff
@@ -28,11 +24,8 @@ describe('My super cool test', () => {
 		await assert.cookie('name').doesNot().exist();
 
 		// test element
-		await assert.element({css: 'div'}).matches({
-			name: PROP.LEFT,
-			type: COMP.LESSER,
-			val: 400,
-		});
+		await assert.element({css: 'div',
```
Comment (andriichumak) [inline demo/test/dummy.test.js]: This is not readable. See our style guide. There is eslint config in the project. Should be:
```
await assert.element({
    css: 'div',
    index: 1,
});
```

Comment (AntonSBilan) [reply demo/test/dummy.test.js]: will fix it, but I guess it is ok with current eslint config, as I understand according to docs https://eslint.org/docs/rules/object-curly-newline we need to remove `"consistent": true` from `"object-curly-newline": ["warn", { "consistent": true }]`

**Pull request merged**